card/mod/05_standard/spec/set/all/history_spec.rb

```diff
@@ -109,7 +109,10 @@
     context 'for subcard' do
       before do
         Card::Auth.as_bot do
-          @card = Card.create name: "left", subcards: {"+right" =>{ content: content}}
+          content = "Nobody expects the Spanish inquisition"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.